endpoint_info.go

```diff
@@ -227,7 +232,17 @@ func (ep *endpoint) DriverInfo() (map[string]interface{}, error) {
 		return nil, fmt.Errorf("failed to get driver info: %v", err)
 	}
 
-	return driver.EndpointOperInfo(n.ID(), ep.ID())
+	epInfo, err := driver.EndpointOperInfo(n.ID(), ep.ID())
+	if err != nil {
+		return nil, err
+	}
+
+	if epInfo != nil {
+		epInfo["GW_INFO"] = gwDriverInfo
+		return epInfo, nil
+	}
+
+	return gwDriverInfo, nil
```

Comment:
The only way to get endpoint attached to a container is to have the daemon pass that information to HCS (the compute service in windows). This relies on querying  HNS ID (a unique id that identifies the endpoint)  from libnetwork for each endpoint connected to sandbox using the DriverInfo call. In an overlay network two endpoints are attached to the sandbox. We need HNS ID for both of them for these endpoints to be actually visible in the container. With code we have right now only gateway endpoint will be visible inside the container